homeassistant/components/xiaomi_aqara.py

```diff
@@ -68,7 +68,7 @@
 
 
 GATEWAY_CONFIG = vol.Schema({
-    vol.Optional(CONF_MAC): GW_MAC,
+    vol.Optional(CONF_MAC, default=None): vol.Any(GW_MAC, None),
```

Comment:
But is `default=None` required?

Reply:
With the new voluptuous, the default value will be passed through the validator. This validator is ok with `None`, so this is ok in this case.

The code that violated this problem the most was code like this: `vol.Optional('bla', default=None): cv.string`. Since `None` is not a valid value for `cv.string`.